docs/security/trust/trust_delegation.md

```diff
@@ -0,0 +1,226 @@
+<!--[metadata]>
++++
+title = "Delegations for content trust"
+description = "Delegations for content trust"
+keywords = ["trust, security, delegations, keys, repository"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Delegations for content trust
+
+Docker Engine supports the usage of the `targets/releases` delegation as the
+canonical source of a trusted image tag.
+
+Using this delegation allows you to collaborate with other publishers without
+sharing your repository key (a combination of your targets and snapshot keys -
+please see "[Manage keys for content trust](trust_key_mng.md)" for more information).
+A collaborator can keep their own delegation key private.
+
+The `targest/releases` delegation is currently an optional feature - in order
+to set up delegations, you must use the Notary CLI:
+
+1. [Download the client](https://github.com/docker/notary/releases) and ensure that it is
+available on your path
+
+2. Create a configuration file at `~/.notary/config.json` with the following content:
+
+	```
+	{
+	  "trust_dir" : "~/.docker/trust",
+	  "remote_server": {
+	    "url": "https://notary.docker.io"
+	  }
+	}
+	```
+
+	This tells Notary where the Docker Content Trust data is stored, and to use the
+	Notary server used for images in Docker Hub.
+
+For more detailed information about how to use Notary outside of the default
+Docker Content Trust use cases, please refer to the
+[the Notary CLI documentation](https://github.com/docker/notary/blob/master/docs/getting_started.md).
```

Comment:
This URL can be changed to `/notary/getting_started/` so that we stay on docs.docker.com. Using links between repositories doesn't work with our tooling (sorry, should've mentioned that)